components/news/Home.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="w-full">
+    <nuxt-link :to="'/news/' + news.slug">
+      <div class="news-image">
+        <img :src="newsImage" :alt="news.title.rendered" class="">
+      </div>
+    </nuxt-link>
+    <div class="grid news-infos">
+      <div class="news-date">
+        <div class="newsgeneral">
+          <nuxt-link to="news/">
+            <h3 class="m-0">
+              News
+            </h3>
+          </nuxt-link>
+        </div>
+        <div class="newsdate">
+          <p>{{ format(new Date(news.modified), 'MM') }}</p>
+          <p>{{ format(new Date(news.modified), 'dd') }}</p>
+        </div>
+      </div>
+      <div class="px-4 news-info-details">
+        <h5 class="py-4">
+          <nuxt-link :to="'/news/' + news.slug">
+            {{ htmlDecoder(news.title.rendered) }}
```

Comment:
I searched our repo for htmlDecoder this is 1000% amazing & smart mixin method! https://github.com/mmmnmnm/lahmacun_rework/blob/03990598f3cee4655f13c949bedfbdb55f74f6ed/plugins/mixinCommonMethods.js#L34 I think you were quite smart to handle it this way

Reply:
Maybe these are not related. 

So I searched for the `htmlDecoder()` stuff and I found that you have a `news/_slug` object and `head()` method there returning the title already what is the difference between that and this, why don't we just use that?

Reply:
htmlDecoder fixes the errors from wordpress json with some extra characters sent as HTML entity codes. you can check what it does if you turn it off in news titles